content/spin/python-components.md

```diff
@@ -328,3 +331,15 @@ redis-cli
 127.0.0.1:6379> get foo
 "bar"
 ```
+
+## Storing Data in the Spin Key-Value Store
+
+Spin has a key-value store built in. For information about using it from Rust, see [the key-value store tutorial](kv-store-tutorial).
+
+## Storing Data in SQLite Relational Databases
+
+For more information about using SQLite relational databases from Python, see [SQLite storage](sqlite-api-guide).
+
+## Storing Data in MySQL and PostgreSQL Relational Databases
```

Comment:
Not supported in Python  as far as I know

Reply:
100% thank you.
This has been updated in new push.